lib/rmt/cli/repos_base.rb

```diff
@@ -38,8 +38,18 @@ def change_repo(id, set_enabled, custom: false)
     puts set_enabled ? _('Repository by ID %{id} successfully enabled.') % { id: id } : _('Repository by ID %{id} successfully disabled.') % { id: id }
   end
 
+  def is_numeric_id?(str)
+    # Check if given string is a plain number without any additional characters
+    # like '15sp3-ptf-repo-id'.
+    !!Integer(str) rescue false
```

Comment:
The `!!` are not doing anything here. Exceptions are not handled like a truthy/falsy value. `Integer(str) rescue false` would be doing the same exact thing.

Reply:
It converts a `integer` into `bool` in the case where `Integer` does not throw. I added it to match the type of the rescue case. Do you think I should remove it?

Reply:
Yeah, It confuses someone else who might have a look. `Integer` throws an exception if it can't be converted.

Reply:
I removed it!